README.md

```diff
@@ -1,15 +1,31 @@
-# Office JavaScript APIs
+# Office Add-ins documentation
 
 Welcome to the Office JavaScript API conceptual documentation. In this repository, you can find the documentation source files for Office JavaScript API concepts, quick starts, tutorials, and how-to guides. For the best experience, we recommend you view this content on [docs.microsoft.com](https://docs.microsoft.com/office/dev/add-ins).
```

Comment:
Suggestion: Given we are changing the title, maybe description should also mention add-ins? I know Office JS API conceptual docs and add-in docs are the same but if I came here without knowing that I might be confused?

Reply:
Re Michelle's suggestion, you could simply change the first sentence to "Welcome to the Office Add-ins documentation repo."